examples/webpack/example-builder.js

```diff
@@ -13,7 +13,7 @@ const baseDir = dirname(fileURLToPath(import.meta.url));
 
 const isCssRegEx = /\.css(\?.*)?$/;
 const isJsRegEx = /\.js(\?.*)?$/;
-const importRegEx = /(?:^|\n)import .* from '(.*)';(?:\n|$)/g;
+const importRegEx = /\simport .* from '([^']+)';\s/g;
```

Comment:
This new regex does not match an import in the first or last line of a file
```suggestion
const importRegEx = /\bimport [^']+ from '([^']+)'/g;
```

As a test:
```js
function match(re, text) {
  let imports = [];
  let m;
  while ((m = re.exec(text))) {
    imports.push(m[1])
  }
  return imports;
}
match(
 	/\bimport [^']+ from '([^']+)'/g,
  `import assert from 'assert';import frontMatter from 'front-matter';
import fs from 'fs'
import fse from 'fs-extra';`
)
```

Reply:
Did you forget to push the fix?

Reply:
Indeed, pushed now. Thanks for the reminder. I used a slightly different regex than the one you suggested, but it passes your test.